### PR TITLE
Feature no longer available in v11

### PR DESCRIPTION
**Pull request by jigneshpshah**
Can be deleted as the feature is no longer available in v11 or specific version can be mentioned in which the feature is available.

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/buying/articles/pull-items-in-purchase-order-based-on-supplier.md [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-
-**Answer:**
-
-To pull items from Material Request for specific Supplier only, follow below given steps.
-
-####Step 1:  Default Supplier
-
-Update Default Supplier in the Item master.
-
-<img alt="Item Purchase UoM" class="screenshot" src="{{docs_base_url}}/assets/img/articles/for-supplier-2.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/buying/articles/pull-items-in-purchase-order-based-on-supplier.md]: also delete the images

Comment (jigneshpshah) [reply erpnext_com/www/docs/user/manual/en/buying/articles/pull-items-in-purchase-order-based-on-supplier.md]: #413

Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/buying/articles/pull-items-in-purchase-order-based-on-supplier.md]: there are two images in the PR, delete both

Comment (jigneshpshah) [reply erpnext_com/www/docs/user/manual/en/buying/articles/pull-items-in-purchase-order-based-on-supplier.md]: Done

**Pull request merged**